Expensify/react-native-live-markdown
# feat: support maxLength prop in web

### Details


This PR includes the implementation of `maxLength` prop in web

### Related Issues


https://github.com/Expensify/App/issues/46766

### Manual Tests


1. Paste this text file (over 400k characters) 
[default.txt](https://github.com/user-attachments/files/17113641/default.txt)

2. Check that the text input in both web and native is cut off to 30k length

### Linked PRs


Pending App PR
